### ISSUE TITLE
📚[DOC]: Update documentation instructions for obtaining ahmed_body dataset

### ISSUE THREAD

**Issue by uribarri** [NVIDIA/modulus-launch#109]
### How would you describe the priority of this documentation request

Medium

### Is this for new documentation, or an update to existing docs?

Update

### Describe the incorrect/future/missing documentation

I would like to get hold of the training dataset for the Modulus CFD ahmed_body example. The README file has instructions for contacting the team at an email address ([simnet-team@nvidia.com](mailto:simnet-team@nvidia.com)) that doesn’t exist anymore, and while there is a sample dataset ([Modulus Datasets: Ahmed Body Aerodynamics | NVIDIA NGC 1](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/modulus/resources/modulus_datasets-ahmed_body_test)) it also says that to get the full dataset I should contact the Modulus team (no contact info provided). Can the documentation be updated to include the current method of getting the dataset?

Posted over on the Modulus forums, and was asked to open an issue here.

**Comment by ram-cherukuri** [comment]
Yes the email address needs to be updated. Here is the contact info to request the data: modulus-team@nvidia.com

**Comment by mnabian** [comment]
Related PR: https://github.com/NVIDIA/modulus-launch/pull/118
